### ISSUE TITLE
can't install v1.2.2 on MacOS with R installed from homebrew

### ISSUE THREAD

**Issue by venpopov** [rstudio/rsconnect#1073]
When I try to install `rsconnect` I get an error because I can't install `PKI`:

```
checking for openssl/ssl.h... no
configure: Missing OpenSSL headers on macOS,
configure: attempting to get headers from Apple...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   257  100   257    0     0   3750      0 --:--:-- --:--:-- --:--:--  3779
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14.8M    0 14.8M    0     0  10.4M      0 --:--:--  0:00:01 --:--:-- 8017k
configure: error: Failed to download OpenSSL sources from Apple. Please install OpenSSL headers before installing PKI.
ERROR: configuration failed for package ‘PKI’
* removing ‘/opt/homebrew/lib/R/4.4/site-library/PKI’
ERROR: dependency ‘PKI’ is not available for package ‘rsconnect’
* removing ‘/opt/homebrew/lib/R/4.4/site-library/rsconnect’
```

I spent 1 hour trying to solve this and here is what I found.

apparently this is a [longstanding](https://github.com/s-u/PKI/issues/17) problem with PKI, which is why `rsconnect` switched to `openssl` package instead back in [2018](https://github.com/rstudio/rsconnect/pull/116).

Unfortunately, the PKI dependency was [reintroduced 2 months ago](https://github.com/rstudio/rsconnect/commit/17f2eff95fbece477dce91b6c341499063077b08)

I can easily install rsconnect 1.2.1 before PKI dependency was reintroduced:

```r
devtools::install_version('rsconnect', version = "1.2.1")
Downloading package from url: https://cloud.r-project.org/src/contrib/Archive/rsconnect/rsconnect_1.2.1.tar.gz
Installing package into ‘/opt/homebrew/lib/R/4.4/site-library’
(as ‘lib’ is unspecified)
* installing *source* package ‘rsconnect’ ...
** package ‘rsconnect’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (rsconnect)
```

**Comment by aronatkins** [comment]
Related: https://github.com/s-u/PKI/issues/32

**Comment by aronatkins** [comment]
@venpopov - Is this still a problem for you now that PKI is installing?

**Comment by venpopov** [comment]
I just checked, now it installs just fine!

**Comment by aronatkins** [comment]
Closing, as availability of PKI was resolved.

**Comment by jonkeane** [comment]
This is a totally separate issue from `rsconnect`, but in case you didn't know this, @venpopov : if you install R from brew with the `--cask` option you will be able to use macOS binaries from CRAN or other repositories. Which would then avoid needing to build things like `PKI` from source.

**Comment by hekatonsure** [comment]
same issue is happening again on Windows, also fixed by installing rsconnect 1.2.1
